sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/DownloadRetryOptions.java

```diff
@@ -30,7 +30,7 @@
      *
      * @return the maximum number of retries to attempt before the request finally fails
      */
-    public int maxRetryRequests() {
+    public int getMaxRetryRequests() {
```

Comment:
nit: `getMaxRetries` or `getMaxRetryAttempts`?

Reply:
I would leave the naming as-is. It presumably has been standardised across languages.